lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -528,17 +529,23 @@ data ByronWalletPutPassphraseData = ByronWalletPutPassphraseData
     , newPassphrase :: !(ApiT (Passphrase "raw"))
     } deriving (Eq, Generic, Show)
 
+newtype ApiTransactionTTL = ApiTransactionTTL
+    { seconds :: NominalDiffTime
+    } deriving (Eq, Generic, Show)
```

Comment:
What about something more consistent with other API fields:

```hs
Quantity "second" NominalDiffTime
```

We have "free" JSON instances for such type already.

Reply:
OK, thanks.